### ISSUE TITLE
ToggleTVSet is not working in the manager on MODX 2.8.4

### ISSUE THREAD

**Issue by pmacswebteam** [Jako/ToggleTVSet#13]
ToggleTVSet 2.0.0-pl and 2.0.2-pl does not work properly in the manger and displays all options.
We have no error in the console, the issue occurs on new resources as well as existing resources, and it _sometimes_ fixes itself when you resize the browser but not always.

**Comment by Jako** [comment]
How did you configure ToggleTVSet?

**Comment by Jako** [comment]
If it fixes itself when the browser is resized, it can't be a configuration issue. It could be a timeout issue in this line: https://github.com/Jako/ToggleTVSet/blob/master/core/components/toggletvset/src/Plugins/Events/OnDocFormPrerender.php#L20

Maybe it helps to increase that timeout.

**Comment by Jako** [comment]
Hmm, the timeout was somehow needed, if you want to toggle an RTE. Maybe it is not needed anymore, since parts of the code are rewritten. I will check this out.

**Comment by pmacswebteam** [comment]
Hi @Jako did you have a chance to look at this?

**Comment by pmacswebteam** [comment]
Hi @Jako we did some testing ourselves and this is what we found.

When the timeout function is entirely removed from OnDocFormPrerender.php, ToggleTVSet stops working.

After that, we tried setting the timeout value to 0 milliseconds and in that case, ToggleTVSet was fixed, similar to what we reported earlier.

Then we decided to test something different by taking the ToggleTVSet.util.init() out of the Modx.on function and placed it inside of a Ext.onReady function and that also fixed our issue.

```
Ext.onReady(function() {
    ToggleTVSet.util.init();
});
```

What are your thoughts on this?
